charts/hono/templates/_helpers.tpl

```diff
@@ -14,9 +14,15 @@
 
 {{/*
 Expand the name of the chart.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+
+The scope passed in is expected to be a dict with keys
+- (mandatory) "dot": the root (".") scope
 */}}
 {{- define "hono.name" -}}
```

Comment:
I think it would be consistent to then also remove the "dot" dict usage in the "hono.name" and "hono.chart" named templates again.